### PR TITLE
API-421: Add "Get product drafts" client resource

### PR DESCRIPTION
**Pull request by damien-carcel**

### CODE REVIEW

File: content/php-client/ee-resources/product-drafts.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ *             ],
+ *             'groups': []
+ *         ],
+ *     ],
+ *     'metadata' => [
+ *         'workflow_status' => 'draft_in_progress',
+ *     ],
+ * ]
+ */
+$publishedProduct = $client->getPublishedProductApi()->get('top');
```
Comment (NolwennP) [inline content/php-client/ee-resources/product-drafts.md]: Wrong

File: content/php-client/ee-resources/product-drafts.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+You can get more information about the returned format of the product values [here](/documentation/resources.html#product-values).
+
+The only difference with a product or a published product is the presence of the last field `metadata`, containing the status of the draft.
```
Comment (NolwennP) [inline content/php-client/ee-resources/product-drafts.md]: Wrong

File: content/php-client/ce-resources/products.md [hunk 3]
```diff
@@ -45,6 +45,8 @@ $product = $client->getProductApi()->get('top');
 
 You can get more information about the returned format of the product values [here](/documentation/resources.html#product-values).
 
+In the Enterprise Edition, the array will contain one more field `metadata`. Look at the [product drafts](/php-client/ee-resources.html#product-draft) for an example.
```
Comment (ahocquard) [inline content/php-client/ce-resources/products.md]: `the response contains one more field`

File: content/php-client/ee-resources/product-drafts.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ *         'workflow_status' => 'draft_in_progress',
+ *     ],
+ * ]
+ */
+$publishedProduct = $client->getProductDraftApi()->get('top');
+```
+
+You can get more information about the returned format of the product values [here](/documentation/resources.html#product-values).
+
+The last field `metadata` (which here contains the status of the draft) is specific to Akeneo PIM enterprise edition.
```
Comment (ahocquard) [inline content/php-client/ee-resources/product-drafts.md]: `Enterprise Edition` (consistency)
`last` => nop, json keys are nor ordered.

```
The field `metadata` is specific to Akeneo PIM Enterprise Edition. 

The status of the draft is specified in this field.
```

**Pull request merged**